pkg/kubelet/eviction/eviction_manager.go

```diff
@@ -578,6 +587,13 @@ func (m *managerImpl) evictPod(pod *v1.Pod, gracePeriodOverride int64, evictMsg
 	m.recorder.AnnotatedEventf(pod, annotations, v1.EventTypeWarning, Reason, evictMsg)
 	// this is a blocking call and should only return when the pod and its containers are killed.
 	klog.V(3).InfoS("Evicting pod", "pod", klog.KObj(pod), "podUID", pod.UID, "message", evictMsg)
+	_, otelSpan := m.tracer.Start(nil, "Eviction/EvictPod", trace.WithAttributes(
+		attribute.String("k8s.pod.uid", string(pod.UID)),
+		attribute.String("k8s.pod", klog.KObj(pod).String()),
+		attribute.String("k8s.pod.name", pod.Name),
+		attribute.String("k8s.namespace.name", pod.Namespace),
+	))
+	defer otelSpan.End()
 	err := m.killPodFunc(pod, true, &gracePeriodOverride, func(status *v1.PodStatus) {
```

Comment:
It would be really nice to pass ctx to killPodFunc, and eventually to the CRI, but we can do that in a follow-up

Reply:
I saw killPodFunc is called in several places out of eviction manager, I think I'd better to solve this in a follow-up pr.